sky/skylet/constants.py

```diff
@@ -137,7 +135,7 @@
     # true.
     '{ bash Miniconda3-Linux-x86_64.sh -b; '
     'eval "$(~/miniconda3/bin/conda shell.bash hook)" && conda init && '
-    f'conda config --set auto_activate_base {conda_auto_activate} && '
+    'conda config --set auto_activate_base {conda_auto_activate} && '
```

Comment:
Can we add a comment here saying that the caller should replace this `conda_auto_activate` with `true` or `false`?

Reply:
Good point! Added. Thanks!